capstone/src/main/webapp/script.js

```diff
@@ -272,11 +273,13 @@ function closePanel() {
   document.getElementById('map').style.width = '100%';
 }
 
+closePanel();
```

Comment:
Should this function call be with other setup code?

Reply:
this is leftover from development, will remove!